### DISCUSSION TITLE
Infection and inflammation

### DISCUSSION THREAD

**Discussion by paolaroncaglia** [monarch-initiative/mondo#3885] [Bigger project]
I was looking at the 'inflammatory disease' branch and I noticed some infection terms.
E.g. MONDO:0000369 'abdominal tuberculosis' http://purl.obolibrary.org/obo/MONDO_0000369 has the axiom:
tuberculosis and (disease has inflammation site some abdomen)
which makes 'abdominal tuberculosis' an inflammatory disease.
This makes sense.
'tuberculosis', however, is not an inflammatory disease in MONDO. I guess that is because inflammatory diseases in MONDO are logically defined based on their location: 
disease or disorder and (disease has inflammation site some anatomical structure)
and that would be harder to state for generic tuberculosis.
So, even though most infectious diseases cause inflammation, I could see why one wouldn't want to generalize.
However, MONDO:0021666 'ear infection' http://purl.obolibrary.org/obo/MONDO_0021666 has the axiom: 
infectious disease and (disease has location some ear)
and the _asserted_ parent 'inflammatory disease'.
Where do you draw the line between infectious diseases and inflammatory ones? An infection may not produce inflammation, e.g. in its very first stage, but once it becomes "visible" and/or is diagnosed, that's often because inflammation is observed. Do I understand correctly that the strategy in MONDO is to link infection to inflammation when an inflamed anatomical location can be established? If so, is this done consistently?
Thanks!

**Comment by cmungall** [comment]
The asserted parent is due to mirroring ncit, but we can override these

I agree it would be good to be more consistent here.

**Comment by nicolevasilevsky** [comment]
How should we go about making this more consistent? Should we remove the equiv axiom that uses 'disease has inflammation site' from:
- 'abdominal tuberculosis'
- 'eye allergy'
- 'gastrointestinal tuberculosis'

Maybe instead use 'disease has location'?

@cmungall

**Comment by nicolevasilevsky** [comment]
@cmungall please see my question above

**Comment by cmungall** [comment]
Yes, change the relation

**Comment by paolaroncaglia** [comment]
@nicolevasilevsky @cmungall 
Thanks. I noticed another one:
MONDO:0021666 ear infection
subclass of
infectious disease of the nervous system
inflammatory disease
disease of ear
It has 5 children, which based on a quick look at their labels seem to be a mix of infectious + inflammatory diseases. If indeed all children are infections as well as inflammations, maybe consider renaming MONDO:0021666 to make that clear? Or create 'ear inflammation' and add as a parent? Sorry, haven't had time to look at all terms in detail. Thank you!

**Comment by nicolevasilevsky** [comment]
@paolaroncaglia re: MONDO:0021666 ear infection:
otitis is an exact synonym, which is inflammation of the ear. 

All of the children terms are infections of some kind.

I feel like this classification make sense. Do you agree?

**Comment by paolaroncaglia** [comment]
@nicolevasilevsky 
Thanks for your feedback. 

I do not disagree with the classification of children of MONDO:0021666 'ear infection', because they are all indeed infections, though for 'otitis externa' this is only clear from the textual definition.
My concern is about the perceived 'exact synonymity' of inflammation and infection terms, in this case, ear infection and ear inflammation ( = otitis). I guess that pretty much all infections, when fully manifesting themselves, have also by then caused inflammation, but there may well be inflammation without the intervention of a pathogen (be it a virus, bacterium, fungus, etc.). 
At the granular level, this may be addressed by not making "-itis" terms exact synonyms of infectious ones, if you and @cmungall agree.

Also, MONDO:0021666 'ear infection' is a Subclass of 'infectious disease of the nervous system'. This doesn't feel entirely correct and may be a similar issue to https://github.com/monarch-initiative/mondo/issues/747?

Thanks again,
Paola

**Comment by nicolevasilevsky** [comment]
Hi @paolaroncaglia 
1) I think changing the synonyms of itis terms would be a big change, I will let @cmungall decide. Some of the synonyms come from other sources, like NCIT, so I think if we wanted to make that change, we'd want to discuss it with the source ontologies.

2) This issue: MONDO:0021666 'ear infection' is a Subclass of 'infectious disease of the nervous system' was addressed in #747, you should see it in the next release.

**Comment by cmungall** [comment]
@paolaroncaglia - I share your concerns. I haven't had time to look fully. To me the most logical solution is always to separate inflammation terms from infection terms, and link with a causal relation where appropriate (from infection to inflammation, if this is an all-some). But for some diseases the conflation may be too deeply part of the medical lexicon.

**Comment by paolaroncaglia** [comment]
@nicolevasilevsky 
@cmungall 
Thanks for your feedback re infection vs. inflammation. I understand the complications due to provenance from other ontologies/resources and/or ingrained medical lexicon. Chris writes he hasn't had time to look fully, so I'll leave it to you to decide if you want to keep this ticket open for further, future discussion, or not. 
(And thanks for fixing the nervous system issue!)

**Comment by cmungall** [comment]
sorry, still have not had time for this
